src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,56 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+export interface Movie {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
+type Cards = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Cards> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = movie;
+
+  return (
```

Comment:
let's de-structure right in params and use an implicit return
```suggestion
export const MovieCard: React.FC<Cards> = ({
    title,
    description,
    imgUrl,
    imdbUrl,
  }) => (
```